### ISSUE TITLE
Validation progress bar display problem

### ISSUE THREAD

**Issue by daihaozxn** [unit8co/darts#1790]
The LSTM model can be trained normally, but it seems that there is a display problem with the progress bar of the validation (the red box in the figure below). ipywidgets is not installed in the virtual environment Pytorch, so it cannot be uninstalled like [#1342.](https://github.com/unit8co/darts/issues/1342) In addition, what does the loss=0.117 in the blue box below refer to?
![darts1](https://github.com/unit8co/darts/assets/73634066/dff253c4-a34d-4ae2-a3c3-9ba01de655a4)

**Comment by madtoinou** [comment]
Hi, would you mind sharing a code snippet to reproduce the issue? This kind of graphical bug/artifacts are often due to the install and I would like to check if it also appears in my virtual environment.

The loss in the blue box should be the training loss, but averaged over the batches whereas the train_loss correspond to the batch-wise loss.

**Comment by daihaozxn** [comment]
Thank you for your reply. The following is mainly the tutorial code in darts, which uses NBEATS to predict electricity data.
It has a similar problem.
```
import torch.nn as nn
import torch.optim
from darts.datasets import ElectricityDataset
import matplotlib.pyplot as plt
from darts.dataprocessing.transformers import Scaler
from darts.models import RNNModel, NBEATSModel
from darts.metrics import mape, smape, mae, rmse

def fit_and_pred(model, training, validation):
    model.fit(series=training, val_series=validation, verbose=True)
    forecast = model.predict(12)
    return forecast

multi_serie_elec = ElectricityDataset().load()
# retaining only three components in different ranges
retained_components = ["MT_002", "MT_008", "MT_009"]
multi_serie_elec = multi_serie_elec[retained_components]
# resampling the multivariate time serie
multi_serie_elec = multi_serie_elec.resample(freq="1H")
# keep the values for the last 5 days
multi_serie_elec = multi_serie_elec[-168:]

# split in train/validation sets
training_set, validation_set = multi_serie_elec[:-48], multi_serie_elec[-48:]

# define a scaler, by default, normalize each component between 0 and 1
scaler_dataset = Scaler()
# scaler is fit on training set only to avoid leakage
training_scaled = scaler_dataset.fit_transform(training_set)
validation_scaled = scaler_dataset.transform(validation_set)

model_NBEATS = NBEATSModel(
    input_chunk_length=24,
    output_chunk_length=12,
    generic_architecture=True,
    num_stacks=10,
    num_blocks=1,
    num_layers=4,
    layer_widths=512,
    expansion_coefficient_dim=5,
    trend_polynomial_degree=2,
    dropout=0.0,
    batch_size=32,
    activation='ReLU',
    nr_epochs_val_period=1,
    model_name="nbeats_run",
    n_epochs=200,
    loss_fn=nn.MSELoss(),
    save_checkpoints=False,
    optimizer_cls=torch.optim.Adam,
    optimizer_kwargs={'lr': 0.001},
    random_state=None,
    pl_trainer_kwargs={
        'accelerator': 'gpu',
        'devices': [0]
    }
)

# training and prediction with the RNN model
forecast_NBEATS = fit_and_pred(model_NBEATS, training_scaled, validation_scaled)
print("RMSE (NBEATS) = {:.4f}".format(rmse(validation_scaled, forecast_NBEATS)))

forecast_NBEATS = scaler_dataset.inverse_transform(forecast_NBEATS)
labels = [f"forecast {component}" for component in retained_components]
fig = plt.figure(figsize=(8, 6))
validation_set.plot()
forecast_NBEATS.plot(label=labels)
# plt.ylim(0, 500)
plt.title("NBEATS model forecast")
plt.legend(loc="upper left")
plt.show()
```

**Comment by madtoinou** [comment]
I could not reproduce the issue, neither on my local install without GPU acceleration nor in a Google colab notebook with access to GPU acceleration.

Could you try reinstalling darts in a new virtual environment and running the notebook again?

**Comment by daihaozxn** [comment]
> 

OK. I will try. Thank you for the suggestions.
